backend/core/src/migration/1633557587028-user-email-lower-case.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class userEmailLowerCase1633557587028 implements MigrationInterface {
+  name = "userEmailLowerCase1633557587028"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
@seanmalbert @YazeedLoonat  Do we only want to change login related email handling here?

We have more columns like %email% in our DB schema:
```
bloom=# SELECT t.table_schema, t.table_name, c.column_name fROM information_schema.tables t INNER JOIN information_schema.columns c on c.table_name = t.table_name and c.table_schema = t.table_schema WHERE c.column_name LIKE '%email%' and t.table_schema not in ('information_schema', 'pg_catalog') and t.table_type = 'BASE TABLE' order by t.table_schema;
 table_schema |    table_name     |     column_name
--------------+-------------------+---------------------
 public       | user_accounts     | email
 public       | listings          | leasing_agent_email
 public       | household_member  | no_email
 public       | applicant         | email_address
 public       | applicant         | no_email
 public       | alternate_contact | email_address
 public       | household_member  | email_address
```

Shouldn't we update those too? E.g. when detecting duplicates in applications (email rule set) by lower casing them we also prevent some problems with missing duplicates.